geetools/ee_dictionary.py

```diff
@@ -8,18 +8,18 @@
 
 @register_class_accessor(ee.Dictionary, "geetools")
 class DictionaryAccessor:
-    """Toolbox for the ``ee.Dictionary`` class."""
+    """Toolbox for the :py:class:`ee.Dictionary` class."""
 
     def __init__(self, obj: ee.Dictionary):
         """Initialize the Dictionary class."""
         self._obj = obj
 
     # -- alternative constructor -----------------------------------------------
-    def fromPairs(self, list: list | ee.List) -> ee.Dictionary:
-        """Create a dictionary from a list of [[key, value], ...]] pairs.
+    def fromPairs(self, list_: list | ee.List) -> ee.Dictionary:
```

Comment:
Even though I understand the though process we cannot afford to change parameter names without a deprecation process to avoid breaking older code so I would prefer to revert it for this PR and keep it for later. and in case of name change maybe we can find something better than a suffixed list (I'm not a huge fan).

Reply:
Ok, I undo this changes in the last commit. Thank you for the comment!